doc/install.txt

```diff
@@ -10,16 +10,12 @@ the local system:
 
 .. _pip: https://pypi.python.org/pypi/pip
 
-``sudo pip install LinkChecker``
-
-Note that this needs the Python development and a C compiler installed.
-For example on Debian Linux systems use ``sudo apt-get install python-dev``.
-
+``sudo pip3 install LinkChecker``
 
 Setup for Windows
 -----------------
-Execute ``LinkChecker-x.y.exe`` and follow the instructions.
-
+Nobody is currently offering an "exe" download for Windows
+See https://github.com/linkchecker/linkchecker/issues/53
```

Comment:
We might suggest `pip install` too, now people will think LinkChecker doesn't work on Windows and won't work until that issue is fixed.

Although... maybe I should boot into Windows and see if linkchecker git actually works or not.

Reply:
I read it that "Setup with pip" was universal and then there were specific suggestions if that didn't work for you. Perhaps that can be emphasised e.g. with headings.

Reply:
People skim.  If I were a Windows user, and I saw a section about installing on Windows, I wouldn't even look at the other text.

Reply:
Perhaps the reality for the foreseeable future is that we are a source only distribution that strongly recommends pip for installation and should just get rid of the mentions of binaries in the install.txt and just have some pointer for how to use pip in different environments?

Reply:
Added a commit with pip instructions for Windows and Mac -  have tried both Windows methods, don't have a Mac.